eng/docgeneration/Generate-DocIndex.ps1

```diff
@@ -134,9 +134,9 @@ function GenerateDocfxTocContent([Hashtable]$tocContent, [String]$lang) {
     Copy-Item "$($RepoRoot)/CONTRIBUTING.md" -Destination "${DocOutDir}/api/CONTRIBUTING.md" -Force
 
     LogDebug "Building site..."
-    & $($DocFx) build "${DocOutDir}/docfx.json"
+    #& $($DocFx) build "${DocOutDir}/docfx.json"
```

Comment:
orthogonal but I'm curious why you don't just remove these and just pass docfx as a parameter?

Reply:
It is better to pass the docfx path in my local run. I am using nuget and npm command to install docfx commands. Have some trouble to find the right path. The best approach is to pass the path as an argument.